autoware_launch/launch/logging_simulator.launch.xml

```diff
@@ -41,14 +40,11 @@
 
   <!-- Vehicle -->
   <group>
-    <let name="launch_vehicle_interface" value="false" if="$(var vehicle_simulation)"/>
-    <let name="launch_vehicle_interface" value="true" unless="$(var vehicle_simulation)"/>
-
     <include file="$(find-pkg-share tier4_vehicle_launch)/launch/vehicle.launch.xml" if="$(var vehicle)">
       <arg name="vehicle_model" value="$(var vehicle_model)"/>
       <arg name="sensor_model" value="$(var sensor_model)"/>
       <arg name="vehicle_id" value="$(var vehicle_id)"/>
-      <arg name="launch_vehicle_interface" value="$(var launch_vehicle_interface)"/>
+      <arg name="launch_vehicle_interface" value="false"/>
```

Comment:
I think you have to make this controllable by an arg for durability tests.

Reply:
[40c34d8](https://github.com/autowarefoundation/autoware_launch/pull/19/commits/40c34d8014cfed2c53887bbfac3012f8d34fe75b)